### DISCUSSION TITLE
remove old virtual builders from builder list

### DISCUSSION THREAD

**Discussion by hmoffatt** [buildbot/buildbot#7778] [Q&A]
I have some builders with virtual builder set that are obsolete and I would like to hide/remove them from the builder list. Is there a way? I don't mind deleting the data permanently from the db if that's what it takes.

**Comment by hmoffatt** [comment]
Just following up on this. Is there a way?

**Reply by p12tic** [reply]
I don't know off the top of my head. You can submit a feature request. 

But I wonder, is disabling "show old builders" not enough?

**Reply by hmoffatt** [reply]
> But I wonder, is disabling "show old builders" not enough?

It's not, the old virtual builders are still shown.

**Reply by hmoffatt** [reply]
I should add that I still have the underlying builder, but I'm not using the virtuals any more. I guess buildbot has no way to know this automatically, hence I'd like to remove them manually.
